client/src/containers/NewsSlider/action.js

```diff
@@ -0,0 +1,42 @@
+import { articleConstants } from './constant';
+import { instance } from '../../axios.instanse';
+
+export const articleActions = {
+  getArticles,
+};
+
+let startArray;
+
+function getArticles() {
+  return dispatch =>{
+    instance.get('/article')
+      .then(res =>{
+        res.data.forEach(element =>{
+          element.text = element.text.slice(0, element.text.indexOf('.'));
+        });
+        if (res.data.length < 4) {
+          startArray = Array.from({ length: res.data.length }, (elem, idx) => idx+1);
+        } else {
+          startArray = Array.from({ length: 4 }, (elem, idx) => idx + 1);
+        }
```

Comment:
const acticleListLength = res.data.length < 4 ? length: res.data.length : 4;
const startArray = Array.from({ length: acticleListLength }, (elem, idx) => idx + 1);

4 in const, pls